test/cali-basic.cpp

```diff
@@ -33,12 +33,14 @@
 // A minimal Caliper instrumentation demo 
 
 #include <Annotation.h>
-
+#include <RichAnnotations.h>
 int main(int argc, char* argv[])
 {
```

Comment:
@khuck we talked in a breakout session about annotations where you make claims about your program, "this should always increase, this should always be less than 2." This is a way we're thinking of implementing that in Caliper.